duckduckgo/Android
# Feature/settings page

Asana Issue URLs: 
- Settings ticket: https://app.asana.com/0/414730916066338/492750258120871
- About Us ticket: https://app.asana.com/0/414730916066338/364465928695401

## Description
Adds a settings page, accessible from the overflow menu on BrowserActivity. 

### Settings Activity
The settings page shows the current app version, and provides a means to navigate to the about us activity.

### About Us Activity
Provides some static text about DDG. Also includes a link to view more information about this, and uses the `BrowserActivity` to navigate to the url if clicked.

## Steps to Test this PR:
1. Use the overflow menu to go to settings
1. Ensure version is shown and correct
1. Click to learn more about DDG
1. Ensure the about us page renders correctly (and is scrollable if the content goes off screen)
1. Click the URL to view more; ensure the app handles the page load (and not any default system browser)
